src/test/java/resources/featurefile/computer.feature

```diff
@@ -0,0 +1,8 @@
+Feature: HomePage Test
+  As user I want to verify page navigation
+
+  @smoke
+  Scenario: User should navigate to Login page successfully
+    Given I am on HomePage page
+    When I click on Computer link
```

Comment:
Hi

Reply:
Hello, Hitesh
I seen your code, looks ok no any impediment on this stage.
Thank you for giving  me code for review.